src/index.js

```diff
@@ -0,0 +1,60 @@
+import './style.css';
+
+const getCityBtn = document.querySelector('button#getCityBtn');
+const cityNameInput = document.querySelector('input#cityName');
+const cityTag = document.querySelector('h1');
+const cityTemp = document.querySelector('p#cityTemp');
+const date = document.querySelector('p#date');
+const tempDescription = document.querySelector('p#tempDescription');
+const tempCBtn = document.querySelector('button#tempCBtn');
+const tempFBtn = document.querySelector('button#tempFBtn');
+const wind = document.querySelector('span#wind');
+const visibility = document.querySelector('span#visibility');
+const humidity = document.querySelector('span#humidity');
+
+function kelvinToCelsius(temp) {
+  return `${(temp - 273.15).toFixed(0)}°C`;
+}
+function kelvinToFarenheit(temp) {
+  return `${((temp - 273.15) * (9 / 5) + 32).toFixed(0)}°F`;
+}
+
+async function getWeatherData() {
```

Comment:
## Replace these function declarations with the ES6 arrow suntax
- [ ] By this time you should be comfortable to use arrow based functions instead of the function keyword when declaring functions. Let's change this to ensure we are adhering to the ES6 syntax as explained in this ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

## Optional Change

- [ ] For purposes of having a modularised codebase, I would advise we move the temperature converter functions (`kelvinToCelsius` & `kelvinToFarenheit`) to a different module, then use ES6's `import` and `export` methods in our respective modules.